### PR TITLE
Allow passing of quantities to altaz helper.

### LINKED ISSUE #248: altaz_to_radec fails when coordinates are given as quantities
`utils.altaz_to_radec` as a hardcoded multiplication by `u.degree` which causes it to fail if the `alt` or `az` are provided as quantity values.  Suggest using `get_quantity_value`. 

Line: https://github.com/panoptes/panoptes-utils/blob/bccd8f963b92b26ad90e30b1a52207b8399aa290/src/panoptes/utils/utils.py#L212

### PR DESCRIPTION
**Pull request by wtgee**
Checks the `alt` and `az` for quantities.

Closes #248

### CODE REVIEW

File: src/panoptes/utils/utils.py [hunk 1]
```diff
@@ -167,26 +167,30 @@ def string_to_params(opts):
     return args, kwargs
 
 
-def altaz_to_radec(alt=35, az=90, location=None, obstime=None, **kwargs):
+def altaz_to_radec(alt=None, az=None, location=None, obstime=None, **kwargs):
```
Comment (danjampro) [inline src/panoptes/utils/utils.py:170]: Why have these as optional args with `None` as default? Would it be better to keep them as required positional arguments without default values?

File: scripts/download-data.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```
Comment (danjampro) [inline scripts/download-data.py:1]: Is this meant to be part of this PR?

File: src/panoptes/utils/utils.py [hunk 3]
```diff
@@ -197,9 +201,9 @@ def altaz_to_radec(alt=35, az=90, location=None, obstime=None, **kwargs):
     AssertionError
 
     Args:
-        alt (int, optional): Altitude, defaults to 35
-        az (int, optional): Azimute, defaults to 90 (east)
-        location (None|astropy.coordinates.EarthLocation, required): A valid location.
+        alt (int or float): Altitude.
```
Comment (danjampro) [inline src/panoptes/utils/utils.py:204]: Or quantity?

File: src/panoptes/utils/utils.py (hunk 1 above)
Comment (wtgee) [reply src/panoptes/utils/utils.py:170]: Mostly for backwards compatibility, although that's probably not required. It is a bit easier to pass things in as a dictionary if it accepts keyword arguments. 

Mostly I am just biased toward keyword args.

File: scripts/download-data.py (hunk 2 above)
Comment (wtgee) [reply scripts/download-data.py:1]: The IERS wasn't downloading without it so the tests were failing.  I changed the description of PR.

**Review by danjampro — APPROVED** [review]
LGTM (apart from `alt=None, az=None`, but I can live with that!).

**Pull request merged**